scripts/newAlias.ts

```diff
@@ -0,0 +1,12 @@
+const aliases = require('../spaces/aliases.json');
+const https = require('https');
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  const oldSpace = args[0];
+  const newSpace = args[1];
+  if (!aliases[oldSpace]) aliases[oldSpace] = newSpace
```

Comment:
```suggestion
  if (aliases[oldSpace]) {
    console.error('Alias already exists for ' + oldSpace)
    process.exit(1);
  }
  
  aliases[oldSpace] = newSpace
```

Let's return an error and fail the whole workflow instead when data are not valid